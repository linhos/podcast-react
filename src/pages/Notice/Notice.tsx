import React from 'react';


const Notice: React.FC = () => {
    return (
        <>
            <h1>Avisos</h1>
        </>
    )
}

export default Notice;