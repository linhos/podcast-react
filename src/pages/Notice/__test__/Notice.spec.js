import React from 'react';
import { render, cleanup, fireEvent, waitFor } from "@testing-library/react";
import {Notice} from '../Notice'

describe('test suite for Notice Page', () => {
    const setup = () => render(<Notice />);
    it('renders', () => {
        const {asFragment} = setup();
        expect(asFragment()).toMatchSnapshot();
    })
    it('should render the Notice title', async() => {
        const {getByText} = setup();
        
        const title = getByText(/Avisos/i);
        expect(title).toBeInTheDocument();
    })
})