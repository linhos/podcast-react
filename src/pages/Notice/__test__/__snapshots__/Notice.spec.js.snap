// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test suite for Notice Page renders 1`] = `
<DocumentFragment>
  <h1>
    Avisos
  </h1>
</DocumentFragment>
`;
